src/components/space-usage/SpaceUsage.js

```diff
@@ -3,7 +3,7 @@ import Chart from "assets/img/illustration/chart.svg";
 
 const SpaceUsage = () => {
   return (
-    <div className="flex flex-wrap items-end content-end justify-center mt-6 relative h-32">
+    <div className="flex flex-wrap items-end content-end justify-center mt-6 relative h-40">
       <img src={Chart} alt="Space usage" className="absolute w-full" />
```

Comment:
Just add class `h-full` after `w-full` and the image will perfectly fit on every screen.
```suggestion
      <img src={Chart} alt="Space usage" className="absolute w-full h-full" />
```